packages/story-editor/src/app/uploader/useUploader.js

```diff
@@ -80,12 +68,19 @@ function useUploader() {
    * Validates a file for upload.
    *
    * @throws Throws an error if file doesn't meet requirements.
-   * @param {Object} file File object.
-   * @param {boolean} canTranscodeFile Whether file can be transcoded by consumer.
-   * @param {boolean} isFileTooLarge Whether file is too large for consumer.
+   * @param {Object} args
+   * @param {Object} args.file File object.
+   * @param {boolean} args.canTranscodeFile Whether file can be transcoded by consumer.
+   * @param {boolean} args.isFileTooLarge Whether file is too large for consumer.
+   * @param {Array} args.overrideAllowedMimeTypes Array of override allowed mime types.
    */
   const validateFileForUpload = useCallback(
-    (file, canTranscodeFile, isFileTooLarge) => {
+    ({
+      file,
+      canTranscodeFile,
+      isFileTooLarge,
+      overrideAllowedMimeTypes = allowedMimeTypes,
```

Comment:
Nit: I'd probably rename this to make less confusing

```suggestion
      allowedMimeTypes = defaultAllowedMimeTypes,
```

Reply:
`allowedMimeTypes` is already declared in the upper scope so we would have to rename quite a few variables.

Reply:
This is why I did it this way. But I am not against changing it.